File: Pyvvo.Logistics/ClientApp/src/app/service/Model/carrier-service.ts
import { Address } from "./Address";
import { Contact } from "./contact";
import { User } from "./user";
import { ShippingMethod } from "./shipping-method";

export class CarrierService {
  id: number;
  createdOn: Date;
  createdOnLocalTime: string;
  updatedOn: Date;
  updateOnLocalTime: string;
  isActive: boolean;
  name: string;
  address: Address;
  contact: Contact;
  createdBy: User;
  shippingMethods: ShippingMethod[];
}

File: Pyvvo.Logistics.Model/Model/CarrierService.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;

namespace Pyvvo.Logistics.Model
{
    public class CarrierService
    {
        [Required, Key] public long Id { get; set; }
        [Required] public long CreatedById { get; set; }
        [Required] public DateTime CreatedOn { get; set; }
        public DateTime UpdatedOn { get; set; }
        public bool IsActive { get; set; }
        public string Name { get; set; }
        public Address Address { get; set; }
        public Contact Contact { get; set; }
        [Required] public User CreatedBy { get; set; }
        public List<ShippingMethod> ShippingMethods { get; set; }
    }
}